scripts/jenkins/dist-feed/dist-feed-manifest.yml

```diff
@@ -120,9 +120,18 @@ packages:
   dist-nilrt-grub_21.8_x64:
     export: "{{ (MNT_NIRVANA_PERFORCEEXPORTS + '/build/exports/ni/nilr/nilrt_onert_system_image/official/export/21.8') | latest_export }}"
     ipk_path: 'targets/linuxU/x64/gcc-4.7-oe/release/onert_system_image_ipk/dist-nilrt-grub_*.ipk'
+  dist-nilrt-grub_22.0_arm:
+    export: "{{ (MNT_NIRVANA_PERFORCEEXPORTS + '/build/exports/ni/nilr/nilrt_onert_system_image/official/export/22.0') | latest_export }}"
+    ipk_path: 'targets/linuxU/armv7-a/gcc-4.7-oe/release/onert_system_image_ipk/dist-nilrt-grub_*.ipk'
   dist-nilrt-grub_22.0_x64:
     export: "{{ (MNT_NIRVANA_PERFORCEEXPORTS + '/build/exports/ni/nilr/nilrt_onert_system_image/official/export/22.0') | latest_export }}"
     ipk_path: 'targets/linuxU/x64/gcc-4.7-oe/release/onert_system_image_ipk/dist-nilrt-grub_*.ipk'
+  dist-nilrt-grub_22.0_x64-combo:
+    export: "{{ (MNT_NIRVANA_PERFORCEEXPORTS + '/build/exports/ni/nilr/nilrt_onert_system_image/official/export/22.0') | latest_export }}"
+    ipk_path: 'targets/linuxU/x64/gcc-4.7-oe/release/onert_system_image_ipk/dist-nilrt-grub_*.ipk'
+  dist-nilrt-grub_22.0_x64-pxi:
```

Comment:
Something's funny here:  In 21.8 we only did the x64 variant and not both the PXI and x64 variants.  Am I missing something?

Reply:
I was basing this on the work item description that says to include these but to use the same ipk as the x64 version. @amstewart - Is that still correct? Or do we only need the two (one x64 and one ARM)?

Reply:
@zraul123
Can you confirm whether the SL reimaging workflows use the `x64-pxi` or `x64-combo` variants of the dist feed IPKs?

If so, we might have mistakenly not exported them with the 21.8 release, and we'll need to update the public feed.

Reply:
Resolved offline with @amstewart. We dropped the `combo` and `pxi` options deliberately and SL will still choose the right image from the two options for a taret.